stories/molecules/newsletter-form.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import NewsletterForm from "components/molecules/NewsletterForm/newsletter-form";
+
+const storyConfig = {
+  title: "Design System/Molecules/NewsletterForm",
+} as ComponentMeta<typeof NewsletterForm>;
+
+export default storyConfig;
+
+const NewsletterFormTemplate: ComponentStory<typeof NewsletterForm> = (args) => <NewsletterForm />;
```

Comment:
I think it would be more productive if we can send some props, to see the state, when the form will be in error state and so on so far